device/mellanox/x86_64-nvidia_sn4280-r0/ACS-SN4280/port_config.ini

```diff
@@ -0,0 +1,51 @@
+##
+## Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+## Apache-2.0
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+
+
+# name          lanes                               alias     index
+Ethernet0       0,1,2,3,4,5,6,7                     etp1      1
+Ethernet8       8,9,10,11,12,13,14,15               etp2      2
+Ethernet16      16,17,18,19,20,21,22,23             etp3      3
+Ethernet24      24,25,26,27,28,29,30,31             etp4      4
+Ethernet32      32,33,34,35,36,37,38,39             etp5      5
+Ethernet40      40,41,42,43,44,45,46,47             etp6      6
+Ethernet48      48,49,50,51,52,53,54,55             etp7      7
+Ethernet56      56,57,58,59,60,61,62,63             etp8      8
+Ethernet64      64,65,66,67,68,69,70,71             etp9      9
+Ethernet72      72,73,74,75,76,77,78,79             etp10     10
+Ethernet80      80,81,82,83,84,85,86,87             etp11     11
+Ethernet88      88,89,90,91,92,93,94,95             etp12     12
+Ethernet96      96,97,98,99,100,101,102,103         etp13     13
+Ethernet104     104,105,106,107,108,109,110,111     etp14     14
+Ethernet112     112,113,114,115,116,117,118,119     etp15     15
+Ethernet120     120,121,122,123,124,125,126,127     etp16     16
+Ethernet128     128,129,130,131,132,133,134,135     etp17     17
+Ethernet136     136,137,138,139,140,141,142,143     etp18     18
+Ethernet144     144,145,146,147,148,149,150,151     etp19     19
+Ethernet152     152,153,154,155,156,157,158,159     etp20     20
+Ethernet160     160,161,162,163,164,165,166,167     etp21     21
+Ethernet168     168,169,170,171,172,173,174,175     etp22     22
+Ethernet176     176,177,178,179,180,181,182,183     etp23     23
+Ethernet184     184,185,186,187,188,189,190,191     etp24     24
+Ethernet192     192,193,194,195,196,197,198,199     etp25     25
+Ethernet200     200,201,202,203,204,205,206,207     etp26     26
+Ethernet208     208,209,210,211,212,213,214,215     etp27     27
+Ethernet216     216,217,218,219,220,221,222,223     etp28     28
+Ethernet224     224,225,226,227,228,229,230,231     etp29     29
+Ethernet232     232,233,234,235,236,237,238,239     etp30     30
+Ethernet240     240,241,242,243,244,245,246,247     etp31     31
+Ethernet248     248,249,250,251,252,253,254,255     etp32     32
```

Comment:
What are the internal ports in this config? Could you add a comment here.

Reply:
This file is not used, latest config is derived from hwsku.json and platform.json, as long as they have Dpc config it's fine.